From me@felipebalbi.com  Mon Jul 27 08:49:32 2009
From: Felipe Balbi <me@felipebalbi.com>
Date: Tue, 21 Jul 2009 15:52:26 +0300
Subject: Staging: cpc-usb: add one todo entry
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Felipe Balbi <me@felipebalbi.com>
Message-ID: <1248180746-18140-5-git-send-email-me@felipebalbi.com>


we need to get rid of those driver-specific error codes

Signed-off-by: Felipe Balbi <me@felipebalbi.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/cpc-usb/TODO |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/cpc-usb/TODO
+++ b/drivers/staging/cpc-usb/TODO
@@ -5,5 +5,6 @@ kernel:
 	- remove proc code
 	- tie into CAN socket interfaces if possible
 	- figure out sane userspace api
+	- use linux's error codes
 
 Send patches to Greg Kroah-Hartman <greg@kroah.com>
